src/bcf/report/table_report/fasta_reader.rs

```diff
@@ -37,11 +37,15 @@ pub fn read_fasta(
     fasta
 }
 
-pub fn get_fasta_length(path: &Path) -> u64 {
+pub fn get_fasta_length(path: &Path, chrom: &str) -> Result<u64, &'static str> {
```

Comment:
Please note as TODO: in the long run, error handling should be solely based on anyhow+thiserror, not custom string based error types.